libs/coin-modules/coin-multiversx/src/signer.ts

```diff
@@ -0,0 +1,13 @@
+export type ElrondAddress = {
+  publicKey: string;
+  address: string;
+};
+
+export type ElrondSignature = {
+  signature: null | Buffer;
+};
+
+export interface ElrondSigner {
+  getAddress(path: string, boolDisplay?: boolean): Promise<ElrondAddress>;
+  sign(path: string, message: string): Promise<ElrondSignature>;
```

Comment:
I think we need to extend this interface with setAddress and provideESDTInfo methods

Reply:
Done